lib/availability_resolver.js

```diff
@@ -68,31 +77,35 @@ class AvailabilityResolver {
     return barcodes
   }
 
-  _fixItemAvailabilityInResponse (barcodesAndAvailability, request) {
+  _checkScsbForRecapCustomerCode () {
+    const item = this.elasticSearchResponse.hits.hits[0]._source.items[0]
+    // If it's an electronic item or a non-recap item, skip it
+    if (item.electronicLocator || !this._isInRecap(item)) Promise.resolve()
+    const barcode = this._getItemBarcode(item)
```

Comment:
Check for null barcode. For example, run the app locally and try this item: http://localhost:8082/api/v0.1/discovery/resources/b11807500-i15959794 Note that it's querying scsb for barcode `null`. Let's not query scsb if there's no barcode.